lib/utils/encoding/sdkSuffix/getSDKSuffix.js

```diff
@@ -0,0 +1,52 @@
+let fs = require('fs');
+let path = require('path');
+let encodeVersion = require('./encodeVersion');
+
+let nodeSDKValue = 'M'; // Constant per SDK
+
+
+/**
+ * @description Turns x.y.z into x.y
+ * @param {string} semVerStr
+ */
+function removePatchFromSemver(semVerStr) {
+  let parts = semVerStr.split('.');
```

Comment:
why not `emVerStr.split('.').slice(0, -1).join('.')` ?

Reply:
The function can accept x.y as well as x.y.z, I've updated the docs to reflect it better.